unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/NFTIcons/NFTTypeIconsAndColors.cs

```diff
@@ -9,23 +9,23 @@ public class NFTTypeIconsAndColors : ScriptableObject
 
     public Sprite GetTypeImage(string nftType)
     {
-        for(int i=0; i< nftIcons.Length; i++)
+        foreach (var t in nftIcons)
         {
-            if(nftIcons[i].key == nftType)
-                return nftIcons[i].value;
+            if(t.key == nftType)
+                return t.value;
         }
 
         return null;
     }
 
     public Color GetColor(string rarity)
     {
-        for(int i=0; i< nftColors.Length; i++)
+        foreach (var t in nftColors)
```

Comment:
Try to avoid to use "only 1 letter" variables and lets use more descriptive names for them. I think it will improve the code readability:
```suggestion
        foreach (var color in nftColors)
```